src/main/resources/application.yaml

```diff
@@ -0,0 +1,80 @@
+#
+# /********************************************************************************
+#  Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+#
+
+server:
+  port: ${APPLICATION_PORT:8080}
+
+spring:
+  threads:
+    virtual:
+      enabled: true
+  application:
+    name: SSI DIM Wallet stub
+
+springdoc:
+  swagger-ui:
+    enabled: ${ENABLE_SWAGGER_UI:true}
+    disable-swagger-default-url: true
+    path: /ui/swagger-ui
+    show-common-extensions: true
+    csrf:
+      enabled: true
+  api-docs:
+    enabled: ${ENABLE_API_DOC:true}
+    path: /docs/api-docs
+
+management:
+  endpoint:
+    loggers:
+      enabled: true
+    health:
+      probes:
+        enabled: true
+
+health:
+  livenessState:
+    enabled: true
+  readinessState:
+    enabled: true
+
+# log level
+logging:
+  level:
+    org:
+      eclipse:
+        tractusx:
+          wallet:
+            stub: ${APP_LOG_LEVEL:DEBUG}
+stub:
+  env: ${STUB_ENV:local}
+  baseWalletBPN: ${BASE_WALLET_BPN:BPNL000000000000} #Oparator BPN
```

Comment:
typo in Operator

Reply:
Fixed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/1042e142d1c22db74eeaf95c02b4b0c50f62f329